pkg/registry/namespace/etcd/etcd_test.go

```diff
@@ -119,7 +126,8 @@ func TestCreateSetsFields(t *testing.T) {
 func TestListEmptyNamespaceList(t *testing.T) {
 	fakeEtcdClient, helper := newHelper(t)
 	fakeEtcdClient.ChangeIndex = 1
-	fakeEtcdClient.Data["/registry/namespaces"] = tools.EtcdResponseWithError{
+	key := path.Join("/", os.Getenv("ETCD_PREFIX"), "/registry/namespaces")
```

Comment:
just to reiterate, we should not see these path.Joins in all the tests, there should be one function in a testetcd package that adjusts for the test prefix.

Additionally, should not have the "/registry" here any more, because that should be added by said function.